[issue-58] - create block for discover holidays

robertnecsoiu opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--jet2holidays-portugal--aemsites.aem.live/
- After: https://issue-58--jet2holidays-portugal--aemsites.aem.page/drafts/rnecsoiu/banner

bstopp reviewed (changes requested):
There is missing padding/margin on the section - shouldn't go full view port width.

Also looks like there's still some missing fonts. Maybe we should put a pin in that or get all of them?

robertnecsoiu commented:
> * https://issue-58--jet2holidays-portugal--aemsites.aem.page/drafts/rnecsoiu/banner

Hey, fixed the padding and some sizes. For the font, for now i've added an import from google fonts, they are getting it also from there, would it be okay for us to also do it?

bstopp commented on blocks/banner/banner.css:
```diff
@@ -0,0 +1,155 @@
+}
+
+.banner.block {
+    --banner-heading-font-family: 'Oswald', sans-serif;
```
If fonts are used in more places, variables should be global, instead of block specific.

bstopp commented on blocks/banner/banner.css:
```diff
@@ -0,0 +1,155 @@
+    --banner-heading-font-family: 'Oswald', sans-serif;
+    --banner-paragraph-font-family: 'Open Sans', sans-serif;
+    --banner-button-font-family: 'Open Sans', arial, sans-serif;
+    --banner-button-font-color: #203240;
```
Same comment about fonts as colors :)

bstopp reviewed (changes requested):
Banner is not quite tall enough. 395px vs the live site's 448 - don't know where the missing px went, but it's visually apparent
